packages/python-pulp-cli-deb/python-pulp-cli-deb.spec

```
@@ -0,0 +1,71 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
+
+# Created by pyp2rpm-3.3.7
+%global pypi_name pulp-cli-deb
+
+Name:           %{?scl_prefix}python-%{pypi_name}
+Version:        0.0.2
+Release:        1%{?dist}
```

```suggestion
Release:        0.1%{?dist}
```

given this is a backport, which needs to be built differently, we need a different NEVR, otherwise Koji will tell us off.

---

Done.